Readme.md

```diff
@@ -29,12 +29,10 @@ dnf install perl-Image-ExifTool
 
 ### Clone the Elodie repository
 
-You can clone Elodie from GitHub. You'll need `git` installed ([instructions](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)).
+You can clone Elodie from GitHub. You'll need `pip` and `git` installed ([instructions](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)).
```

Comment:
Can we also add a link to install pip since that's a new dependency?

```
You can clone Elodie from GitHub. You'll need `pip` and `git` installed ([git instructions](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) and [pip instructions](https://pip.pypa.io/en/stable/installing/)).
```